packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -245,6 +245,9 @@ class CupertinoDatePicker extends StatefulWidget {
   /// positive integer factor of 60.
   ///
   /// [use24hFormat] decides whether 24 hour format is used. Defaults to false.
+  ///
+  /// [dateOrder] determines the order of the columns inside [CupertinoDatePicker] in date mode.
+  /// Defaults to [localizations.datePickerDateOrder].
```

Comment:
`[localizations]` does not exist I think? Maybe just say defaults to the locale's default date format/order?